packages/svgs/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "@hopper-ui/svgs",
+    "author": "Workleap",
+    "version": "0.0.0",
+    "description": "The svgs package.",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/gsoft-inc/wl-hopper.git",
+        "directory": "packages/svgs"
+    },
+    "publishConfig": {
+        "access": "public",
+        "provenance": true
+    },
+    "type": "module",
+    "files": [
+        "/dist",
+        "CHANGELOG.md",
+        "README.md"
+    ],
+    "scripts": {
+        "build": "node --loader ts-node/esm src/scripts/write-icons.ts",
+        "test": "jest --config jest.config.ts"
+    },
+    "dependencies": {
+        "@hopper-ui/styled-system": "workspace:*"
+    },
+    "devDependencies": {
+        "@testing-library/jest-dom": "6.1.4",
```

Comment:
a lot of those dependencies will need to be cleanup, lots of react/dom stuff

Reply:
it won't be needed for testing? I suppose some of the testing is done directly in the code, but maybe we still need tests using testing library?

Reply:
i'm not 100% sure to be honest, we can cleanup after we've written some tests